src/Ghcid.hs

```diff
@@ -263,8 +265,9 @@ runGhcid session waiter termSize termOutput opts@Options{..} = do
 
 
 -- | Given an available height, and a set of messages to display, show them as best you can.
-prettyOutput :: String -> Int -> [Load] -> [(Style,String)]
-prettyOutput currTime loaded [] = [(Plain,allGoodMessage ++ " (" ++ show loaded ++ " module" ++ ['s' | loaded /= 1] ++ ", at " ++ currTime ++ ")")]
-prettyOutput _ loaded xs = concat $ msg1:msgs
+prettyOutput :: String -> Int -> Bool -> [Load] -> [(Style,String)]
+prettyOutput currTime loaded _ [] = [(Plain,allGoodMessage ++ " (" ++ show loaded ++ " module" ++ ['s' | loaded /= 1] ++ ", at " ++ currTime ++ ")")]
+prettyOutput _ loaded removeBold xs = concat $ msg1:msgs
```

Comment:
Instead of wiring `removeBold` all the way inside `prettyOutput`, I suggest you have `termOutput` take a boolean saying color or not, and remove all styles if it is False.